week4-ts/src/styles/theme.ts

```diff
@@ -0,0 +1,22 @@
+import { DefaultTheme } from 'styled-components';
+const fontSizes = {
+  form: '40px',
```

Comment:
```suggestion
import { DefaultTheme } from 'styled-components';
const fontSizes = {
  form: css`
    font-size: 40px;
  `,
```

사이즈만 있다면 폰트사이즈 다 선언해줘도 좋겠다 ~~

Reply:
무슨차이지?!! 가져다가 쓸때는 똑가티
```ts
theme.fontSizes.form
```
아닌,,,,가,,,?

Reply:
`font-size: $({( theme )} => theme.fontSizes.form};` 으로 쓸 거를
`$({( theme )} => theme.fontSizes.form};` 으로 쓸 수 있다 차이랄ㄹ까!